packages/next/client/head-manager.ts

```diff
@@ -79,7 +79,7 @@ function updateElements(type: string, components: JSX.Element[]): void {
     }
   )
 
-  oldTags.forEach((t) => t.parentNode!.removeChild(t))
+  oldTags.forEach((t) => t?.parentNode?.removeChild(t))
```

Comment:
```suggestion
  oldTags.forEach((t) => t.parentNode?.removeChild(t))
```

I don't think `t` can be null/undefined anymore